lib/fauna/errors.rb

```diff
@@ -91,8 +72,97 @@ class InternalError < FaunaError; end
   # An exception thrown if FaunaDB responds with an HTTP 503.
   class UnavailableError < FaunaError; end
 
+  # :section: ErrorData
+
+  # Data for one error returned by the server.
+  class ErrorData
+    def self.from_hash(hash)
+      code = hash[:code].to_sym
```

Comment:
I was going to say no, but those reference keys that would (had no error occurred) come back as symbols, so that should actually probably stay